library/pom.xml

```diff
@@ -310,6 +310,11 @@
       <artifactId>commons-beanutils</artifactId>
       <version>1.8.3</version>
     </dependency>
+    <dependency>
+      <groupId>net.sf.trove4j</groupId>
+      <artifactId>trove4j</artifactId>
+      <version>3.0.3</version>
+    </dependency>
   </dependencies>
```

Comment:
http://trove4j.sourceforge.net/html/license.html
We can't use trove4j because of GPL